cfssl/assets/service_checks.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "agent_version": "7.7.0",
```

Comment:
Just curious, how did you get `7.7.0`?

Reply:
Let's put 7.36.0, since this PR's `pyproject.toml` seems to depend on `datadog-checks-base>=25.1.0`, which comes included starting on [Agent 7.36.0](https://github.com/DataDog/integrations-core/blob/7.36.x/datadog_checks_base/CHANGELOG.md).